.github/workflows/ubuntu-workflow.yml

```diff
@@ -28,6 +28,12 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
+            
+      - name: Gradle check
+        uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-executable: "./test_runner/gradlew"
+          arguments: "-p test_runner check"
```

Comment:
the same comment about `build` and `check`

you could check it using `--dry-run`